### PR TITLE
Update old grid to new grid

### PR DESCRIPTION
**Pull request by maya**
...with the exception of:

- components/grid - redirects to the v2 grid
- components/color - redirects to v2 utilities/color

[😎 Preview](https://federalist-proxy.app.cloud.gov/preview/uswds/uswds-site/update-grid-v2)

### CODE REVIEW

File: _data/nav.yml [hunk 1]
```diff
@@ -23,9 +23,11 @@ Components:
   - text: Overview
     href: /components/
   - href: /components/typography/
-  - href: /components/colors/
+  - text: Colors
+    href: /components/colors/
   - href: /components/icons/
-  - href: /components/grids/
+  - text: Grids
+    href: /components/grids/
```
Comment (maya) [inline _data/nav.yml]: Another option is we could also change this to the `utilities/*` links so it's not a redirect and goes directly. Thoughts?

File: _includes/banner.html [hunk 2]
```diff
@@ -2,7 +2,7 @@
 <section class="usa-banner site-banner">
```
Comment (maya) [inline _includes/banner.html]: Can eventually use the v2 banner + `site-banner` class.

File: _data/nav.yml (hunk 1 above)
Comment (thisisdano) [reply _data/nav.yml]: I might suggest moving colors out of components altogether

**Comment by thisisdano** [comment]
I don't think I'm seeing the redirects working.

**Comment by maya** [comment]
Ah, OK. They were working before, it might be bc I force added the titles in 08ce2d3.

File: _data/nav.yml (hunk 1 above)
Comment (maya) [reply _data/nav.yml]: Removed "Colors" in c750d57.

Comment (maya) [reply _data/nav.yml]: Keep grids in components?

**Comment by maya** [comment]
@thisisdano redirects should be working now. Take a look now.

**Pull request merged**